oe_contact_forms.module

```diff
@@ -215,8 +216,16 @@ function _oe_contact_forms_clear_block_cached_definitions(): void {
  */
 function oe_contact_forms_contact_form_presave(EntityInterface $entity) {
   _oe_contact_forms_clear_block_cached_definitions();
-  // Invalidate cache tags if config is changed.
-  Cache::invalidateTags($entity->getCacheTags());
+  // Invalidate form display cache tags if config is changed,
+  // since the EntityFormDisplay is not in storage we have to instantiate here.
+  $entity_form_display = new EntityFormDisplay([
```

Comment:
is it really need to instantiate a new entity form display to get its cache tags?

Reply:
refactored to a simple string